src/roktManager.ts

```diff
@@ -77,6 +84,12 @@ export default class RoktManager {
     }
 
     public selectPlacements(options: IRoktSelectPlacementsOptions): Promise<IRoktSelection> {
+        if (this.experimentAllocation) {
+            options.attributes['rokt.experimentid'] = this.experimentAllocation.experimentId;
+            options.attributes['rokt.bucketid'] = this.experimentAllocation.bucketId;
+            options.attributes['rokt.userid'] = this.experimentAllocation.userId;
+        }
+
```

Comment:
You should put this in the try/catch block below. The queuing logic should be the first thing in this function because it runs before the rokt web sdk and kit initialize.